src/js/extensions/toolbar.js

```diff
@@ -469,12 +467,21 @@ var Toolbar;
                 manualStateChecks.push(extension);
             });
 
+            parentNode = Selection.getSelectedParentElement(selectionRange);
+
+            // Make sure the selection parent isn't outside of the contenteditable
+            if (!this.getEditorElements().some(function (element) {
```

Comment:
The previous loop was verifying that `parentNode` had a tag name, which would prevent the code from passing the element into the extensions if the `parentNode` was either a text node, or `document` (which apparently doesn't have a `tagName` either).  Since I've remove the `tagName` check, make sure the `parentNode` is either an editor element, or a descendant of one, then we don't have to worry about hitting the `document`